pkg/monitor/cluster/cluster.go

```diff
@@ -135,6 +135,7 @@ func (mon *Monitor) Monitor(ctx context.Context) (errs []error) {
 		mon.emitStatefulsetStatuses,
 		mon.emitJobConditions,
 		mon.emitSummary,
+		mon.emitEtcdMetrics,
```

Comment:
In addition to this, we could also push metrics about node cpu utilization and memory utilization. If we start pulling metrics from prometheus and push them to geneva, then we might as well take those.